pkg/k8shandler/status.go

```diff
@@ -16,6 +18,11 @@ import (
 const healthUnknown = "cluster health unknown"
 const NOT_FOUND_INDEX = -1
 
+var DISK_WATERMARK_LOW_PCT *float64
```

Comment:
I will mention, but I prefer the style as implemented here, varnames including constant code style in golang are always camelcase I believe.